web/src/components/molecules/Schema/ModelFieldList.tsx

```diff
@@ -32,7 +48,7 @@ const ModelFieldList: React.FC<Props> = ({
           actions={[
             <Icon
               icon="delete"
-              onClick={() => handleFieldDelete((item as Field).id)}
+              onClick={() => handleFieldDeleteConfirmation((item as Field).id)}
```

Comment:
Just a suggestion, you can change line 102 to `const FieldStyledList = styled(List<Field>)` and then:
```suggestion
              onClick={() => handleFieldDeleteConfirmation(item.id)}
```

Reply:
@nourbalaha , there is an issue with `antd` it can't predict the type I'm passing to `renderItem`